docs/torch/link_pred.rst

```diff
@@ -102,82 +102,89 @@ Our goal is to create a model capable of predicting whether an edge exists betwe
 
 .. code-block:: python
 
-	>>> from dataclasses import dataclass
-	>>> import argparse
-	>>> import numpy as np
-	>>> import torch
-	>>> from deepgnn.pytorch.modeling.base_model import BaseModel
-	>>> from deepgnn.graph_engine import SamplingStrategy, GEEdgeSampler, GraphEngineBackend
-	>>> from deepgnn.pytorch.common.utils import set_seed
-	>>> from deepgnn.pytorch.common.dataset import TorchDeepGNNDataset
-	>>> from deepgnn.pytorch.modeling import BaseModel
-	>>> from deepgnn.pytorch.training import run_dist
-	>>> from deepgnn.pytorch.common.metrics import F1Score
+    >>> from typing import Dict
+    >>> from dataclasses import dataclass
+    >>> import tempfile
```

Comment:
unused import.

Reply:
All 3 of these are used.

Reply:
Once you address comment below about saving model, then this import will not be needed.